prow/cluster/horologium_deployment.yaml

```diff
@@ -35,6 +35,7 @@ spec:
         image: gcr.io/k8s-prow/horologium:v20190125-2aca69d
         args:
         - --job-config-path=/etc/job-config
+        - --dry-run=false
```

Comment:
Does this flag exist in `gcr.io/k8s-prow/horologium:v20190125-2aca69d`?

We need to make this flag default to true for some time period before we require people to set it.

Reply:
you mean make the flag default to false for some period?